Fix javadoc warnings in telemetry-http-java11

The java11 http module has javadoc warnings when running `./gradlew javadoc` that we should clean up:

```
/Users/jplumb/code/newrelic-telemetry-sdk-java/telemetry-http-java11/src/main/java/com/newrelic/telemetry/Java11Http.java:11: warning: no @return
  public static TelemetryClient newTelemetryClient(String apiKey) {
                                ^
/Users/jplumb/code/newrelic-telemetry-sdk-java/telemetry-http-java11/src/main/java/com/newrelic/telemetry/Java11Http.java:21: warning: no @return
  public static TelemetryClient newTelemetryClient(BaseConfig baseConfig) {
                                ^
2 warnings
```

--- a/telemetry-http-java11/src/main/java/com/newrelic/telemetry/Java11Http.java
+++ b/telemetry-http-java11/src/main/java/com/newrelic/telemetry/Java11Http.java
@@ -7,6 +7,7 @@ public class Java11Http {
    * Create a new TelemetryClient with the given apiKey and default configuration
    *
    * @param apiKey - The New Relic insert api key
+   * @return a telemetry client
    */
   public static TelemetryClient newTelemetryClient(String apiKey) {
     return TelemetryClient.create(Java11HttpPoster::new, apiKey);
@@ -17,6 +18,7 @@ public class Java11Http {
    * Everything not covered in the baseConfig will be defaulted.
    *
    * @param baseConfig - A configuration instance with the basics
+   * @return a telemetry client
    */
   public static TelemetryClient newTelemetryClient(BaseConfig baseConfig) {
     return TelemetryClient.create(Java11HttpPoster::new, baseConfig);

